examples/dialin-chatbot/bot_daily.py

```diff
@@ -87,6 +141,44 @@ async def on_first_participant_joined(transport, participant):
     async def on_participant_left(transport, participant, reason):
         await task.queue_frame(EndFrame())
 
+    @transport.event_handler("on_dialin_ready")
+    async def on_dialin_ready(_, sip_endpoint):
+        logger.info(f"on_dialin_ready: {sip_endpoint}")
+
+    @transport.event_handler("on_dialin_connected")
+    async def on_dialin_connected(transport, event):
+        logger.info(f"on_dialin_connected: {event}")
+        sip_session_id = event["sessionId"]
+
+        async def transfer_call(
+            function_name, tool_call_id, args, llm: LLMService, context, result_callback
+        ):
+            logger.debug(f"transfer_call: {function_name} {tool_call_id} {args}")
+
+            # sip_url = "sip:your_user_name@sip.linphone.org"
+
+            sip_url = (
+                f"sip:your_username@dailyco.sip.twilio.com?x-daily_id={room_url.split('/')[-1]}"
```

Comment:
this sip uri needs to come from the developer. It will be different.

Reply:
What @vr000m said.  `x-daily_id` is not currently used.
sip_uri comes from the response to adding sip related configuration to the daily room.

Reply:
This is a custom parameter that we are sending to Twilio, so that the receiving SIP agent can pull the transcript for that daily room, based on the x-daily-id.